### ISSUE TITLE
No way to trigger job pane commands from menus

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#3744]
(This is a preview of many issues to come with accessibility push in 1.3.)

The open-source jobs pane contains two commands, "Start Job" and "Clear Jobs", but these are not reachable from the main menu. Perhaps could be added to a new `Tools / Jobs / ...` entry, similar to Terminal.

Presumably "Start Job" would redisplay the Jobs pane if it is currently hidden.
